doc/howto/network_increase_bandwidth.md

```diff
@@ -46,3 +69,6 @@ To do this, use one of the following methods:
 
 - Apply the same changes as described above for the LXD host.
 - Set the `queue.tx.length` device option on the instance profile or configuration.
+  For example, to do this for the LXD default profile:
+
+       sudo lxc profile device set default eth0 queue.tx.length "10000" 
```

Comment:
You shouldn't need sudo for this command? (If your user is in the `lxd` group, which we assume for most of the docs.)

Reply:
Thanks for letting me know, I didn't realize the docs assumed the user was part of lxd group. Will keep this in mind for any other doc edit pull requests.